src/selinux_overview.md

```diff
@@ -14,12 +14,6 @@ Note: When SELinux is installed, there are three well defined directory
 locations referenced. Two of these will change with the old and new
```

Comment:
This "Note" paragraph above should be removed as well.

Reply:
Huh, could have sworn I'd deleted that paragraph too, but this is what I get for sending patches late at night. Fixed.